std/assembly/typedarray.ts

```diff
@@ -34,6 +50,14 @@ export class Uint8ClampedArray extends TypedArray<u8,u32> {
   subarray(begin: i32 = 0, end: i32 = 0x7fffffff): Uint8ClampedArray {
     return changetype<Uint8ClampedArray>(super.subarray(begin, end));
   }
```

Comment:
plz use one line separator between methods

Reply:
Alright. Is this documented somewhere? tslint didn't pick it up.

Reply:
just missing rule. But just observe rest sources and you found this using everywhere